features/steps/steps.go

```diff
@@ -140,8 +141,23 @@ func (c *Component) iShouldReceiveAnErrorsArray() error {
 	return nil
 }
 
+//we are passing the string array as [xxxx,yyyy,zzz]
+//this is required to support array being used in kafka messages
+func arrayParser(raw string) (interface{}, error) {
+	//remove the starting and trailing brackets
+	str := strings.Trim(raw, "[]")
+	if str == "" {
+		return []string{}, nil
+	}
+
+	strArray := strings.Split(str, ",")
+	return strArray, nil
+}
+
 func (c *Component) theFollowingExportStartEventsAreProduced(events *godog.Table) error {
-	expected, err := assistdog.NewDefault().CreateSlice(new(event.ExportStart), events)
+	assist := assistdog.NewDefault()
```

Comment:
Is there a slightly better place to put this kind of configuration? 
This is more related to test setup, and not step definition